### ISSUE TITLE
VS2017RC3 - MSB3274 Higher version than the currently targeted framework

### ISSUE THREAD

**Issue by JamesNK** [dotnet/roslyn-project-system#1493]
I have a project Newtonsoft.Json with these target frameworks:

```<TargetFrameworks>net45;net40;net35;net20;netstandard1.0;portable-net45+win8+wpa81+wp8</TargetFrameworks>```

I have a project Newtonsoft.Json.Tests that has target frameworks:

```<TargetFrameworks>net40</TargetFrameworks>```

and references the first project via:

```<ProjectReference Include="..\Newtonsoft.Json\Newtonsoft.Json.csproj" />```

I get this warning:

> warning MSB3274: The primary reference "C:\Development\Source\Newtonsoft.Json\Src\Newtonsoft.Json\bin\Debug\Net45\Newtonsoft.Json.dll" could not be resolved because it was built against the ".NETFramework,Version=v4.5" framework. This is a higher version than the currently targeted framework ".NETFramework,Version=v4.0".

Newtonsoft.Json.Tests should use the net40 output from Newtonsoft.Json

**Comment by JamesNK** [comment]
Nevermind - I was referencing the wrong project.
